### PR TITLE
fix(chat): fix big space and 3 dots in title (Issue #1494)

### PR DESCRIPTION
**Pull request by Derikyan**
**Description:**

Big spaces between filters and conversations sections
Publication request title is cut without 3 dots and without tooltip

Issues:

- Issue #1494 

**UI changes**

Before:
Снимок экрана 2024-06-13 в 20 41 46

After:
Снимок экрана 2024-06-13 в 20 46 03
Снимок экрана 2024-06-13 в 20 47 03
Снимок экрана 2024-06-13 в 20 47 39

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1582.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1582.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by Derikyan** [comment]
/deploy-review
>- **Deployment failed**: [canceled](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/486406)

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1582.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1582.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1582.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by Derikyan** [comment]
/deploy-review
>- **Deployment failed**: [canceled](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/486881)

File: apps/chat/src/components/Chat/Publish/PublishWizard.tsx [hunk 1]
```diff
@@ -416,7 +416,7 @@ export function PublishModal({
             entities={entities}
             path={path}
             files={files}
-            containerClassNames="px-3 py-4 md:px-5"
+            containerClassNames="px-3 py-4 md:px-5 overflow-auto"
```
Comment (IlyaBondar) [inline apps/chat/src/components/Chat/Publish/PublishWizard.tsx]: I don't think that we want to allow horizontal scroll
```suggestion
            containerClassNames="px-3 py-4 md:px-5 overflow-y-auto"
```

File: apps/chat/src/components/Chat/Publish/PublishWizard.tsx [hunk 2]
```diff
@@ -331,15 +331,15 @@ export function PublishModal({
...
+        'group/modal flex min-h-[579px] min-w-full max-w-[1100px] !bg-layer-2 md:h-[747px] md:min-w-[550px] lg:min-w-[1000px] xl:w-[1100px]',
         { 'w-full': files.length },
       )}
       dataQa="publish-modal"
       state={isOpen ? ModalState.OPENED : ModalState.CLOSED}
       onClose={onClose}
       initialFocus={nameInputRef}
     >
-      <div className="flex h-full flex-col divide-y divide-tertiary">
+      <div className="flex flex-col divide-y divide-tertiary overflow-auto">
```
Comment (IlyaBondar) [inline apps/chat/src/components/Chat/Publish/PublishWizard.tsx]: I don't think that we want to allow horizontal scroll
```suggestion
      <div className="flex flex-col divide-y divide-tertiary overflow-y-auto">
```

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1582.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**